tests/test_ontodoc_rst.py

```diff
@@ -0,0 +1,20 @@
+"""Test ontodoc"""
+
+
+# if True:
+def test_ontodoc():
+    """Test ontodoc."""
+    from pathlib import Path
+
+    from ontopy import get_ontology
+    from ontopy.ontodoc_rst import OntologyDocumentation
+    import owlready2
+
+    emmo = get_ontology("https://w3id.org/emmo/1.0.0-rc1").load()
```

Comment:
I am not so happy about tests that dowload EMMO. We already have many of those and they make the tests very slow.

Reply:
Replaced it with the local animal ontology.